libnethack_client/src/clientapi.c

```diff
@@ -993,6 +993,61 @@ nhnet_get_topten(int *out_len, char *statusbuf, const char * volatile player,
 }
 
 
+struct nh_board_entry *
+nhnet_get_board_entries(char *entries_since, int *length)
+{
+    struct nh_board_entry *brdlist;
```

Comment:
Initialize your locals, even if you reinitialize them later. This prevents accidentally using uninitialized variables.